src/react-apps/ux-editor/src/selectors/getFormData.ts

```diff
@@ -0,0 +1,87 @@
+import { createSelector, createSelectorCreator, defaultMemoize } from 'reselect';
+const isEqual = require('lodash.isequal');
+
+const formDataSelector = (state: IAppState) => {
+  return state.formFiller.formData;
+};
+
+const formDataForContainerSelector = (state: IAppState, props: any) => {
+  const layout = state.formDesigner.layout;
+  const componentsInContainer = Object.keys(layout.components).filter(
+    (componentId: string) => {
+      return layout.order[props.id].indexOf(componentId) > -1;
+    },
+  );
+
+  const container = layout.containers[props.id];
+  const filteredFormData: any = {};
+
+  componentsInContainer.forEach((componentId: string) => {
```

Comment:
"for in/of" loops would make this more tidy.

Reply:
Updated to for of loop